client/src/components/auth/ForgotPassword.vue

```diff
@@ -0,0 +1,180 @@
+        <p class="info-text">{{ lang.ENTER_YOUR_EMAIL_ADDRESS }}</p>
+        <VForm>
+            <VCol cols="11" sm="11" md="8" class="pa-0 py-4">
+                <label>{{ lang.EMAIL }}*</label>
+                <VTextField
+                    id="email"
+                    :value="email"
+                    :error-messages="emailErrors"
+                    @input="setEmailOnInput"
+                    @blur="setEmail"
+                    outlined
+                    dense
+                    class="rounded"
+                />
+            </VCol>
+            <VCol cols="11" sm="11" md="8" class="pa-0">
+                <VRow no-gutters align="center" justify="space-between">
+                    <VBtn
+                        height="44"
+                        class="login-button  primary"
+                        depressed
+                        @click="onSubmit"
+                        >{{ lang.SEND_RESET_INSTRUCTION }}
+                    </VBtn>
+                    <RouterLink
+                        :to="{ name: 'SignIn' }"
+                        class="remembered-password "
+                    >
+                        {{ lang.REMEMBERED_PASSWORD }}
+                    </RouterLink>
+                </VRow>
+            </VCol>
+            <VSpacer class="pa-4" />
+            <VCol cols="11" sm="11" md="8" class="pa-0">
+                <VAlert
+                    :type="typeResultSubmitResetPassword"
+                    dense
+                    outlined
+                    text
+                    dismissible
+                    :value="helperVisibility"
+                >
+                    <h6>{{ resultOfSubmitResetPassword }}</h6>
+                    <p>{{ explanation }}</p>
+                </VAlert>
+            </VCol>
+        </VForm>
+    </div>
+</template>
+
+<script>
+import * as actions from '@/store/modules/auth/types/actions';
+import { mapActions } from 'vuex';
+import enLang from '@/store/modules/i18n/en';
+import { validationMixin } from 'vuelidate';
+import { required, email } from 'vuelidate/lib/validators';
+import * as notificationActions from '@/store/modules/notification/types/actions';
+
+export default {
+    name: 'ForgotPassword',
+    mixins: [validationMixin],
+    components: {},
+    data: () => ({
+        lang: enLang,
+        email: '',
+        resultSubmitResetPassword: '',
+        typeResultSubmitResetPassword: '',
+        explanation: '',
+        helperVisibility: false
+    }),
+    validations: {
+        email: { required, email }
+    },
+    methods: {
+        ...mapActions('auth', {
+            forgotPassword: actions.FORGOT_PASSWORD
+        }),
+        ...mapActions('notification', {
+            setErrorNotification: notificationActions.SET_ERROR_NOTIFICATION
+        }),
+        setEmail(e) {
+            this.email = e.target.value;
+            this.$v.email.$touch();
+        },
+        setEmailOnInput(e) {
+            this.email = e;
```

Comment:
probably e.target.value?

Reply:
event Input work in Vuetify in that strange way it returns value of field not object.

Reply:
then why didn't you call it `value`?)

Reply:
done